content/20.treatments.md

```diff
@@ -788,63 +713,41 @@ Furthermore, no conclusions about correlations between the severity of disease a
-The Coalition for Epidemic Preparedness Innovations (CEPI) is coordinating global health agencies and pharmaceutical companies to develop vaccines against SARS-CoV-2.
-As of April 8, 2020, there were 115 vaccine candidates to prevent COVID-19, of which 78 were active.
-Of the 78 active vaccine programs, 73 were in the preclinical or exploratory stage [@doi:10.1038/d41573-020-00073-5].
+##### Vaccine Development
 
-Historically, an H1N1 influenza vaccine was developed relatively efficiently, mainly because influenza-vaccine technology had already been developed and regulatory agencies had already decided that vaccines produced using egg- and cell-based platforms could be licensed under the regulations used for a strain change.
-Critiques of the experience producing and distributing the H1N1 vaccine have stressed the need for alternative development-and-manufacturing platforms that can be readily adapted to new pathogens.
-Although a monovalent H1N1 vaccine was not available before the pandemic peaked in the United States and Europe, it was available soon afterward as a stand-alone vaccine that was eventually incorporated into the commercially available seasonal influenza vaccines [@doi:10.1056/NEJMp2005630].
+Flu-like illnesses caused by viruses are a common target of vaccine development, and influenza vaccine technology in particular has made many strides.
+During the H1N1 influenza outbreak, vaccine development was accelerated because of the existing infrastructure, along with the fact that regulatory agencies had already decided that vaccines produced using egg- and cell-based platforms could be licensed under the regulations used for a strain change.
+Critiques of the production and distribution of the H1N1 vaccine have stressed the need for alternative development-and-manufacturing platforms that can be readily adapted to new pathogens.
+Although a monovalent H1N1 vaccine was not available before the pandemic peaked in the United States and Europe, it was available soon afterward as a stand-alone vaccine that was eventually incorporated into commercially available seasonal influenza vaccines [@doi:10.1056/NEJMp2005630].
 If H1N1 vaccine development provides any indication, considering developing and manufacturing platforms for promising COVID-19 vaccine trials early could hasten the emergence of an effective prophylactic vaccine against SARS-CoV-2.
 
+The first critical step towards developing a vaccine against SARS-CoV-2 was characterizing the target, which fortunately happened early in the COVID-19 outbreak with the sequencing and dissemination of the viral genome [@url:https://www.who.int/csr/don/12-january-2020-novel-coronavirus-china/en/].
+The Coalition for Epidemic Preparedness Innovations (CEPI) is coordinating global health agencies and pharmaceutical companies to develop vaccines against SARS-CoV-2.
+As of April 8, 2020, there were already 115 vaccine candidates against prevent SARS-CoV-2, of which 78 were under active development.
```

Comment:
Do we need an update here?

Reply:
Hmm I'm not sure where this came from. In the data source we use for Figure 2, there seem to be 116. I'm not sure how they're defining "under active development" because that reports whether they're recruiting and what phase they're in. I'm going to adjust it to be more vague for this PR, but I should be able to edit the code so that it can automatically update these numbers!
Data source: <http://covid19.trialstracker.net/>